game-core/build.gradle

```diff
@@ -0,0 +1,373 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'org.junit.platform:junit-platform-gradle-plugin:1.0.2'
+    }
+}
+
+plugins {
+    id 'java'
+    id 'application'
+    id 'checkstyle'
+    id 'jacoco'
+    id 'com.github.ben-manes.versions' version '0.17.0'
+    id 'com.github.johnrengelman.shadow' version '2.0.2'
+    id 'com.install4j.gradle' version '7.0.1'
+    id 'de.undercouch.download' version '3.3.0'
+    id 'net.ltgt.errorprone' version '0.0.13'
+}
+
+apply from: '../gradle/scripts/yaml.gradle'
+apply plugin: 'org.junit.platform.gradle.plugin'
+
+group = 'triplea'
+description = 'TripleA is a free online turn based strategy game and board game engine, similar to such board games as Axis & Allies or Risk.'
+mainClassName = 'games.strategy.engine.framework.GameRunner'
+
+ext {
+    artifactsDir = file("$buildDir/artifacts")
+    releasesDir = file("$buildDir/releases")
+    remoteLibsDir = file('.remote-libs')
+    rootFilesDir = file("$buildDir/rootFiles")
+    schemasDir = file('config/triplea/schemas')
+
+    gameEnginePropertiesFile = file('game_engine.properties')
+    gameEnginePropertiesArtifactFile = file("$rootFilesDir/${gameEnginePropertiesFile.name}")
+}
+
+checkstyle  {
+    configFile = file("$rootProject.projectDir/config/checkstyle/checkstyle.xml")
```

Comment:
Can this just be added to the existing `checkstyle` task configuration on line 331?  (Note that it must be defined before the `configProperties` property, which makes use of the `configFile` property.)

Reply:
Looks to not be a problem, good recommendation: 01062ba